lib/ramble/ramble/test/repository.py

```diff
@@ -12,18 +12,20 @@
 import ramble.paths
 
 
-@pytest.fixture()
-def extra_repo(tmpdir_factory):
+@pytest.fixture(params=["applications", "", "foo"])
```

Comment:
Is it easy to add an end_to_end style test too? I don't love that this adds a user exposed API/CLI surface (`-d`) and doesn't explicitly use it

Reply:
This should be fixed. I added some tests in `cmd/repo.py` to actually use this feature.